src/fsharp/NicePrint.fs

```diff
@@ -1736,7 +1764,7 @@ module private TastDefinitionPrinting =
 
 //--------------------------------------------------------------------------
 
-module private InferredSigPrinting = 
```

Comment:
As there're no assemblies in JS, only files. Fable doesn't make a distinction between `internal` and `public`, so using internal modules should work.